### PR TITLE
Feature/ltbi results processing

### PR DESCRIPTION
**Pull request by yaqi7**
Here is the updated script which changed CI to percentile, assign 0 to NA (NA from 0 denominator), and merged 100 and 10 file together. Please let me know if there is any question. Thank you
https://github.com/ihmeuw/vivarium_csu_ltbi/blob/feature/ltbi_results_processing/src/vivarium_csu_ltbi/tools/ltbi_result_process.py

### CODE REVIEW

File: src/vivarium_csu_ltbi/tools/number_and_percent_under5_with_hh_AcTB.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import pandas as pd, numpy as np
+from db_queries import get_population
+
+def mergeCountry(df1, df2):
+    if "random_seed" in df1.columns:
+        del df1['random_seed']
+    if "random_seed" in df2.columns:
+        del df2['random_seed']
+    df = pd.merge(df1.reset_index(), df2.reset_index(), on = ["input_draw_number", "random_seed", "ltbi_treatment_scale_up.scenario"], how = 'outer')
```
Comment (aflaxman) [inline src/vivarium_csu_ltbi/tools/number_and_percent_under5_with_hh_AcTB.py]: Let's change this line together.  I still have a different idea on how to combine df1 and df2.

**Review by aflaxman — CHANGES_REQUESTED** [review]

File: src/vivarium_csu_ltbi/tools/ltbi_result_process.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    df_100_incidence = shape(df_total, pattern)
+    u5_calculation(df_100_incidence, data, country, measure, "U5_hh")
+    hiv_calculation(df_100_incidence, data, country, measure, "PLWHIV")
+    return
+
+
+if __name__ == '__main__':
+    result_dir = '/ihme/costeffectiveness/results/vivarium_csu_ltbi/'
+    loc_100_filePath = {
+    'ethiopia_end_100': result_dir + 'updated-input-data-end-100/ethiopia/2020_01_03_14_22_51/output.hdf',
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/tools/ltbi_result_process.py:81]: there are more recent results than this now

**Comment by halfhorst** [comment]
I'm going to work on incorporating this into the results processing command so that it is automatically generated with the other tables. I'll tag you both on the PR when I'm done

**Comment by yaqi7** [comment]
> I'm going to work on incorporating this into the results processing command so that it is automatically generated with the other tables. I'll tag you both on the PR when I'm done

Thank you Cody, I am updating the script using add function to replace merge and will push the updated one in a few minutes.

**Comment by yaqi7** [comment]
> I'm going to work on incorporating this into the results processing command so that it is automatically generated with the other tables. I'll tag you both on the PR when I'm done

@halfhorst Thank you Cody, I did a little change of replacing merge function with add function. Following is the most recent version:
https://github.com/ihmeuw/vivarium_csu_ltbi/blob/bee0f3d3769280d32e4eb8ae44dc66b49d48ec76/src/vivarium_csu_ltbi/tools/ltbi_result_process.py

**Comment by halfhorst** [comment]
I believe this is covered by #101, and the new make_results produces 'averted' information for the final tables as well

**Pull request closed without merging**